pydal/helpers/classes.py

```diff
@@ -488,7 +488,8 @@ def try_create_web2py_filesystem(db):
                 raise NotImplementedError(
                     "DatabaseStoredFile only supported by mysql, potresql, sqlite"
                 )
-            sql = "CREATE TABLE IF NOT EXISTS web2py_filesystem (path VARCHAR(255), content BLOB, PRIMARY KEY(path));"
+            blobType = "BYTEA" if db._adapter.dbengine == "postgres" else "BLOB" 
+            sql = "CREATE TABLE IF NOT EXISTS web2py_filesystem (path VARCHAR(255), content %(blobType)s, PRIMARY KEY(path));" % {"blobType": blobType}
```

Comment:
This change is problematic. It will may break apps of current users if the table already exists. Can you explain why it is necessary?

Reply:
The "BLOB" field type doesn't exist in PostgreSQL, it's type "BYTEA", so currently it just errors attempting to create the web2py_filesystem table with "BLOB type does not exist" since it's not available in PostgreSQL. 
[https://www.postgresql.org/docs/current/datatype-binary.html](https://www.postgresql.org/docs/current/datatype-binary.html)